packages/legend-application-studio/src/components/workspace-setup/WorkspaceSetup.tsx

```diff
@@ -57,13 +60,91 @@ import {
 import { debounce, guaranteeNonNullable } from '@finos/legend-shared';
 import { WorkspaceSetupStore } from '../../stores/workspace-setup/WorkspaceSetupStore.js';
 import { DocumentationLink } from '@finos/legend-lego/application';
+import { openShowcaseManager } from '../../stores/ShowcaseManagerState.js';
 
 const WorkspaceSetupStoreContext = createContext<
   WorkspaceSetupStore | undefined
 >(undefined);
 
 export const DEFAULT_WORKSPACE_SOURCE = 'HEAD';
 
+export const ShowcaseCard: React.FC = () => {
+  const applicationStore = useLegendStudioApplicationStore();
+  return (
+    <BaseCard
+      className="workspace-setup__content__card"
+      cardMedia={undefined}
+      cardName="Showcase projects"
+      cardContent="Review showcase projects with sample project code and re-use existing code snippets to quickly build your model."
+      cardActions={[
+        {
+          title: 'Showcase explorer',
+          content: <FileImportIcon />,
+          action: () => openShowcaseManager(applicationStore),
+        } as MuiCardActionConfig,
+      ]}
+      isStable={true}
+    />
+  );
+};
+
+export const SandboxCard: React.FC = () => {
+  const applicationStore = useLegendStudioApplicationStore();
+  const sandboxDocument = applicationStore.documentationService.getDocEntry(
+    LEGEND_STUDIO_DOCUMENTATION_KEY.APPLICATION_SANDBOX,
+  );
+  return (
+    sandboxDocument && (
+      <BaseCard
+        className="workspace-setup__content__card"
+        cardMedia={undefined}
+        cardName={sandboxDocument.title!}
+        cardContent={sandboxDocument.markdownText!.value}
+        cardActions={[
+          {
+            title: sandboxDocument.text!,
```

Comment:
why do have `sandboxDocument.text!` as the title?

Reply:
Since some card contents are not open source, I utilized the class below to render them.


```
export class DocumentationEntry {
  readonly key!: string;

  markdownText?: MarkdownText | undefined;
  title?: string | undefined;
  text?: string | undefined;
  url?: string | undefined;
  related?: string[] | undefined;
  ......
}
```
In this case 

- `markdownText` -> `Review showcase projects with sample project code and re-use existing code snippets to quickly build your model.`
- `title` -> `SHOWCASE EXPLORER`
- `text` ->  `Showcase projects`

<img width="783" alt="Screenshot 2023-12-06 at 11 51 24 AM" src="https://github.com/finos/legend-studio/assets/73408381/3d2a92b8-e62e-476d-9bc8-9729a99fc42d">


Re: why do have sandboxDocument.text! as the title?
It's just an option to choose which field is used to have what info. I could switch `title` to have card name

Reply:
p.s. internal code is waiting for review too

Reply:
my concern was using `!` here without knowing whether the field was indeed present.